nuget/lib/dependabot/nuget/metadata_finder.rb

```diff
@@ -12,7 +12,73 @@ class MetadataFinder < Dependabot::MetadataFinders::Base
       def look_up_source
         return Source.from_url(dependency_source_url) if dependency_source_url
 
-        look_up_source_in_nuspec(dependency_nuspec_file)
+        src_repo = look_up_source_in_nuspec(dependency_nuspec_file)
+        return src_repo if src_repo
+
+        # Fallback to getting source from the search result's projectUrl or licenseUrl.
+        # GitHub Packages doesn't support getting the `.nuspec`, switch to getting
+        # that instead once it is supported.
+        src_repo_from_project
+      rescue StandardError
+        # At this point in the process the PR is ready to be posted, we tried to gather commit
+        # and release notes, but have encountered an exception. So let's eat it and log it
+        # since it's better to have a PR with no info than error out.
+        # TODO how do we log this?
```

Comment:
We don't have a pattern for logging / metrics in core right now as we don't have a way to inject our backend services for either into core.

This is something we should add to improve observability though, but for now swallowing an error mysteriously is consistent behaviour 😅